lib/src/Control/Arrow/DefaultError.hs

```diff
@@ -0,0 +1,16 @@
+{-# LANGUAGE Arrows #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE UndecidableInstances #-}
+module Control.Arrow.DefaultError where
+
+import Control.Arrow
+import Control.Arrow.Fail
+
+class ArrowFail e c => ArrowDefaultError e c where
```

Comment:
What does this type class do? It is not clear from the name or the type. Please comment that describes what it does.